config/routes/spree.rb

```diff
@@ -85,6 +85,7 @@
         get :invoice
         get :print
         get :print_ticket
+        put :update_note
```

Comment:
@mkllnk : not sure this is the right way to go. This adds:`admin/orders/ORDER_ID/update_note` and act as an API (does not render any html)

Wondering where you would put this one?

Thanks! 🙏

Reply:
We are just updating in attribute on the order model. So in conventional Rails that would be handled by the `#update` method. I can see why that may not seem the right fit here. The update method does a lot more and it uses flash with redirects for the response. One way could be to respond to html with the current logic and respond to json with your new logic. The other option would be to add an `update` method to the API v0 OrdersController to update an order. That might actually be cleaner.

That Spree routes thing is outdated and I think that we should slowly replace it. Speaking of replacing, wouldn't this be a nice example for Reactive Rails? Haha, I'm probably overcomplicating this and blowing out the scope.

Reply:
> The other option would be to add an update method to the API v0 OrdersController to update an order. That might actually be cleaner.

Yep, let's go! (I didn't want to modify the api/v0, but seems to be a good idea)